### PR TITLE
Fix strings not marked for translation

### PR DESCRIPTION
**Pull request by nlhkabu**
Found while doing #7927

### CODE REVIEW

File: warehouse/templates/manage/account.html [hunk 1]
```diff
@@ -437,7 +437,8 @@ <h2 class="sub-title">{% trans %}Change password{% endtrans %}</h2>
         </ul>
       </div>
       <div>
-        <input value="Update password" class="button button--primary" type="submit" data-target="password-match.submit">
+        {% set updateButtonText=gettext("Update password") %}
+        <input value="{{ updateButtonText }}" class="button button--primary" type="submit" data-target="password-match.submit">
```
Comment (di) [inline warehouse/templates/manage/account.html:441]: Just curious, why use `gettext` over `trans` here?

```suggestion
        <input value="{% trans %}Update Password{% endtrans %}" class="button button--primary" type="submit" data-target="password-match.submit">
```

Comment (nlhkabu) [reply warehouse/templates/manage/account.html:441]: I have no idea :tired_face: :laughing:

Comment (di) [reply warehouse/templates/manage/account.html]: OK, fine with me either way I think. If you want to accept this suggestion you'll likely need to run `make translations` again though.

**Comment by nlhkabu** [comment]
Thanks @di.
Translations updated. This should be ready to merge.

**Pull request merged**